tests/test_lockfile_pattern/pixi.toml

```
@@ -0,0 +1,45 @@
+[project]
+name = "pixi-pack"
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "osx-64", "linux-64", "linux-aarch64", "win-64"]
+
+[tasks]
+build = "cargo build --release"
+test = "cargo test"
+
+[dependencies]
+rust = "==1.77.2"
+openssl = "3.*"
+pkg-config = "*"
+k9s = "*"
+
+[target.linux.dependencies]
+compilers = ">=1.7.0"
+
+[feature.test.dependencies]
+conda = "*"
+micromamba = "*"
+
+[feature.demo.dependencies]
+vhs = "*"
+[feature.demo.tasks]
+demo = "vhs .github/assets/demo/demo-light.tape && vhs .github/assets/demo/demo-dark.tape"
+
+[feature.lint.dependencies]
+pre-commit = "*"
+prettier = "*"
+taplo = "*"
+pre-commit-hooks = "*"
+typos = "*"
+[feature.lint.tasks]
+pre-commit-install = "pre-commit install"
+pre-commit-run = "pre-commit run -a"
+
+[environments]
+default = ["lint", "test"]
+demo = { features = ["demo"], no-default-feature = true }
+lint = { features = ["lint"], no-default-feature = true }
+
```

```suggestion
[project]
name = "my-project"
channels = ["conda-forge"]
platforms = ["linux-64"]

```

let's make this a bit simpler

---

de4094cb1b7b2b5219ab438bd63a824e7c2b2324